woodwork/type_sys/inference_functions.py

```diff
@@ -93,8 +94,18 @@ def boolean_nullable_func(series):
                 {False},
             ]:
                 return True
+            series_lower = set(str(s).lower() for s in set(series))
```

Comment:
should we have this be optional based on an input param? Not 100% convinced that's the best approach but curious on your thoughts

Reply:
Also if there is a `nan` value here, it will be stringified and then lowercased. In that event, it won't match any of the pairs in `config.get_option("boolean_inference_strings")`. Is this intentional?